altair/api.py

```diff
@@ -282,6 +284,32 @@ def encode(self, **kwargs):
         self.encoding = Encoding(**kwargs)
         return self
 
+    def configure(self, **kwargs):
+        """Set chart configuration"""
+        self.vlconfig = VLConfig(**kwargs)
+        return self
+
+    def set_single_dims(self, width, height):
```

Comment:
Maybe put this logic in a `_height_changed` and `_width_changed` method of the VLConfig class so the other things get updated automatically always. After playing with this, do you think a user will ever want to fine tune this stuff? Seems confusing enough that maybe not? We should definitely bring with up with Jeff.